### PR TITLE
ARROW-7628: [Python] Clarify docs of csv reader skip_rows and nulls in strings

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

File: python/pyarrow/_csv.pyx [hunk 1]
```diff
@@ -109,8 +109,8 @@ cdef class ReadOptions:
     @property
     def skip_rows(self):
         """
-        The number of rows to skip at the start of the CSV data, not
-        including the row of column names (if any).
+        The number of rows to skip at the start of the CSV data (the row
+        of column names (if any) does not need to be included in this count).
```
Comment (pitrou) [inline python/pyarrow/_csv.pyx]: Perhaps "The number of rows to skip before the column names (if any) and the CSV data." ?

Comment (pitrou) [reply python/pyarrow/_csv.pyx]: @nealrichardson What do you think?

Comment (jorisvandenbossche) [reply python/pyarrow/_csv.pyx]: Ah, I like your suggestion (that should also make it clear, without needing my wordy after note)

Comment (nealrichardson) [reply python/pyarrow/_csv.pyx]: Yeah I like @pitrou's phrasing.

**Pull request closed without merging**